### PR TITLE
(MODULES-2179) changes to add support for SetEnvIfNoCase

### PR DESCRIPTION
**Pull request by shanmer**
Add support for directive SetEnvIfNoCase as per ticket https://tickets.puppetlabs.com/browse/MODULES-2179

### CODE REVIEW

**Comment by shanmer** [comment]
Added section to README.md file

File: spec/acceptance/vhost_spec.rb [hunk 1]
```diff
@@ -1135,8 +1135,21 @@ class { 'apache': }
         host { 'test.server': ip => '127.0.0.1' }
         apache::vhost { 'test.server':
           docroot  => '/tmp',
-          setenv   => ['TEST /test'],
-          setenvif => ['Request_URI "\.gif$" object_is_image=gif']
+          setenv         => ['TEST /test'],
+          setenvif       => ['Request_URI "\.gif$" object_is_image=gif']
```
Comment (hunner) [inline spec/acceptance/vhost_spec.rb]: Why are these lines changed?

File: manifests/vhost.pp [hunk 2]
```diff
@@ -636,7 +641,7 @@
   # - $proxy_pass_match
   # - $proxy_preserve_host
   # - $no_proxy_uris
-  if $proxy_dest or $proxy_pass or $proxy_pass_match or $proxy_dest_match {
+  if $proxy_dest or $proxy_pass or $proxy_pass_match  or $proxy_dest_match {
```
Comment (mhaskel) [inline manifests/vhost.pp]: This has an extra space that should be removed.

Comment (hunner) [inline manifests/vhost.pp]: Why is there a space here?

**Comment by hunner** [comment]
Could you rebase this?

**Comment by hunner** [comment]
@shanmer Ping on this again :)

**Comment by tphoney** [comment]
Hi @shanmer, thanks for the work you have put into this pull request. However due to inactivity and  needs rebased and squashed this pull request is being closed. If you want to re-submit this PR with the necessary changes, we will gladly look at the new pull request.
